atuin/src/command/client/history.rs

```diff
@@ -303,6 +307,12 @@ impl Cmd {
 
         print_list(&history, mode, format.as_deref());
```

Comment:
Looks like this is printing the list twice if reverse == true.

Reply:
My bad, just realised it. Changing this..